radix-engine-interface/src/blueprints/resource/worktop.rs

```diff
@@ -1,17 +1,44 @@
 use crate::blueprints::resource::*;
+use crate::constants::RESOURCE_MANAGER_PACKAGE;
 use crate::data::scrypto::model::*;
 use crate::math::Decimal;
 use crate::*;
+use radix_engine_common::data::scrypto::*;
 use radix_engine_common::types::*;
 use sbor::rust::prelude::*;
+use sbor::*;
 
 pub const WORKTOP_BLUEPRINT: &str = "Worktop";
 
 pub const WORKTOP_DROP_IDENT: &str = "Worktop_drop";
 
 #[derive(Debug, Eq, PartialEq, ScryptoSbor)]
 pub struct WorktopDropInput {
-    pub worktop: Own,
+    pub worktop: OwnedWorktop,
+}
+
+#[derive(Debug, Eq, PartialEq, ScryptoCategorize, ScryptoEncode, ScryptoDecode)]
+#[sbor(transparent)]
+pub struct OwnedWorktop(pub Own);
+
+impl Describe<ScryptoCustomTypeKind> for OwnedWorktop {
+    const TYPE_ID: GlobalTypeId =
+        GlobalTypeId::Novel(const_sha1::sha1("OwnedWorktop".as_bytes()).as_bytes());
+
+    fn type_data() -> Option<TypeData<ScryptoCustomTypeKind, GlobalTypeId>> {
```

Comment:
Ahh I see what you mean about this - at some point I wanted to add an `#[sbor(validation_getter = get_validation_method_name)]` attribute to the schema generator for the Describe trait 👍